src/Settings/Default.hs

```diff
@@ -267,6 +267,9 @@ disableWarningArgs = builder Ghc ? mconcat
                                     , "-Wno-tabs" ]
       , libraryPackage       ? pure [ "-Wno-deprecated-flags" ] ] ]
 
+defaultGhcPkgArgs :: Args
```

Comment:
According to current conventions, `defaultGhcPkgArgs` should live in `Settings/Packages/GhcPkg.hs`.

Reply:
right